android/app/src/main/java/net/minetest/minetest/MainActivity.java

```diff
@@ -114,6 +124,31 @@ private void startNative() {
 		startActivity(intent);
 	}
 
+	@RequiresApi(Build.VERSION_CODES.O)
+	private void createNotificationChannel() {
+		NotificationManager notifyManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+		if (notifyManager == null)
+			return;
+
+		String channelId = "net.minetest.minetest";
```

Comment:
ideally this would be a `public static final` so the magic value exists only once
but doesn't matter too much

Reply:
Done.